src/Nethermind/Nethermind.Core/Crypto/Keccak.cs

```diff
@@ -48,6 +52,17 @@ public static ValueHash256 Compute(string input)
             return InternalCompute(System.Text.Encoding.UTF8.GetBytes(input));
         }
 
+        private readonly struct BytesWrapper(byte[] bytes) : IEquatable<BytesWrapper>
+        {
+            internal readonly byte[] _bytes = bytes;
```

Comment:
Would it be better if it was in-line array of 32 (and maybe additional byte for length)?

Reply:
Is just a pointer copy/wrapper; if it was an inline array it would be an additional copy?

If it was 32 + 1 (33 bytes) it would also be multiple instructions to copy vs 8 bytes and also allocating for the `byte[]` path